notebooks/requirements_dev.txt

```diff
@@ -25,7 +25,7 @@ pytest==7.0.1
 pytest-mock==3.7.0
 pytest-mypy==0.9.1
 pytest-runner==6.0.0
-Sphinx==4.4.0
+Sphinx==4.3.2
```

Comment:
Sphinx 4.4.0 is not compatible with the latest version of flake8:
```
   The conflict is caused by:
      build 0.7.0 depends on importlib-metadata>=0.22; python_version < "3.8"
      flake8 4.0.1 depends on importlib-metadata<4.3; python_version < "3.8"
      pytest 7.0.1 depends on importlib-metadata>=0.12; python_version < "3.8"
      sphinx 4.4.0 depends on importlib-metadata>=4.4; python_version < "3.10"
```
using the proper / non-legacy pip resolver prevents dependabot from making invalid version upgrades in the future.